.github/workflows/ci.yml

```diff
@@ -20,25 +20,33 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        variant:
-          - fakemachine-arch
-          - fakemachine-bookworm
-          - fakemachine-bullseye
+        os: [bullseye, bookworm]
+        backend: [qemu, uml]
```

Comment:
If there's a way, I would add a comment why kvm isn't in the list. Similarly for the Arch below (as mentioned offline UML is not officially supported in Arch just yet)